generated/nidaqmx/constants.py

```diff
@@ -394,15 +401,15 @@ class InputCalSource(Enum):
 
 
 class InputDataTransferCondition(Enum):
-    ON_BOARD_MEMORY_MORE_THAN_HALF_FULL = 10237  #: Transfer data from the device when more than half of the onboard memory of the device fills.
-    ON_BOARD_MEMORY_NOT_EMPTY = 10241  #: Transfer data from the device when there is data in the onboard memory.
-    ONBOARD_MEMORY_CUSTOM_THRESHOLD = 12577  #: Transfer data from the device when the number of samples specified with **ai_data_xfer_custom_threshold** are in the device FIFO.
-    WHEN_ACQUISITION_COMPLETE = 12546  #: Transfer data when the acquisition is complete.
+    ONBRD_MEM_MORE_THAN_HALF_FULL = 10237  #: Transfer data from the device when more than half of the onboard memory of the device fills.
+    ONBRD_MEM_NOT_EMPTY = 10241  #: Transfer data from the device when there is data in the onboard memory.
+    ONBRD_MEM_CUSTOM_THRESHOLD = 12577  #: Transfer data from the device when the number of samples specified with **ai_data_xfer_custom_threshold** are in the device FIFO.
+    WHEN_ACQ_COMPLETE = 12546  #: Transfer data when the acquisition is complete.
 
 
 class InvertPolarity(Enum):
-    NO = 0
-    YES = 1
+    DO_NOT_INVERT_POLARITY = 0  #: Do not invert polarity
+    INVERT_POLARITY = 1  #: Invert polarity
```

Comment:
This is fine, but YES and NO are a bit nicer, if we can easily fix it.

Reply:
Updated to `YES`, `NO`